cluster/charts/crossplane/templates/crossplane-admin-clusterrole.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.personas.deploy }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ template "name" . }}-admin
+  labels:
+    app: {{ template "name" . }}
```

Comment:
Would it make sense to use the `app.kubernetes.io/name` label?

https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Reply:
We've already got `-lapp=crossplane` in use in this repo.

After a quick bit of surveying, I'm not sure why `app` is as prevalent as it is.  There are alternatives such as `application` and `k8s-app`.  With such diversity, I'm more inclined to adopt the recommended label.  In my local `kind` `core-dns` deployment, `k8s-app` is used.  Tiller and [this nginx example in the Kubernetes docs](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set) use `app`.

Since that change would have to affect existing docs and tools, I would like to defer on adopting the recommended label.